From 22721031e2a729ddc145312af618d8cfbb3fa647 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Thu, 8 Jul 2021 18:29:00 -0400
Subject: [PATCH 5/5] scarlet: Don't wake on pen events

This is because it is causing more issues than good. A simple bump on
the stylus hangar will wake the tablet.

ChromeOS handles this differently, AFAIUI, by having some form of
userspace suspend tooling that can do more fancy things at wake.
---
 arch/arm64/boot/dts/rockchip/rk3399-gru-scarlet.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-gru-scarlet.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-gru-scarlet.dtsi
index 88b3b0f84ea2e..69b399d1ec2ba 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-gru-scarlet.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-gru-scarlet.dtsi
@@ -508,7 +508,6 @@ camera: &i2c7 {
 		gpios = <&gpio1 1 GPIO_ACTIVE_LOW>;
 		linux,code = <SW_PEN_INSERTED>;
 		linux,input-type = <EV_SW>;
-		wakeup-source;
 	};
 };
 
-- 
2.29.2

